rmf_traffic_editor/gui/delete_dialog.cpp

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (C) 2019 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#include "delete_dialog.h"
+#include <algorithm>
+#include <QtWidgets>
+using std::vector;
+using std::string;
+
+
+DeleteDialog::DeleteDialog(
+  QWidget* parent,
+  Building& building,
```

Comment:
Can this be changed to `const Building&`? This dialog shouldn't change the building